html/contactus.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Us- Ethanol Detection</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-image: url('contactus.jpg');
+            /* Replace 'your-background-image.jpg' with your actual image file */
+            background-size: cover;
+            background-position: center;
+        }
+
+        .container {
+            max-width: 800px;
+            margin: 50px auto;
+            padding: 20px;
+            background-color: rgba(255, 255, 255, 0.582);
+            /* Add a semi-transparent white background */
+           
+	    border-radius: 8px;
+            box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+            display: flex;
+            justify-content: space-between;
+            flex-wrap: wrap;
+        }
+
+        .left-column {
+            width: calc(50% - 20px);
+        }
+
+        .right-column {
+            width: calc(50% - 20px);
+        }
+
+        h1 {
+            text-align: center;
+            color: #333;
+            width: 100%;
+        }
+
+        .contact-info {
+            padding: 20px;
+            background-color: #f4f4f4;
+            border-radius: 8px;
+            margin-bottom: 20px;
+        }
+
+        .contact-info h2 {
+            color: #333;
+            margin-bottom: 10px;
+        }
+
+        .contact-info p {
+            color: #666;
+            margin-bottom: 5px;
+            display: flex;
+            align-items: center;
+        }
+
+        .map-responsive {
+            padding-bottom: 56.25%;
+            position: relative;
+            height: 0;
+            margin-bottom: 20px;
+        }
+
+        .map-responsive iframe {
+            left: 0;
+            top: 0;
+            height: 100%;
+            width: 100%;
+            position: absolute;
+        }
+
+        .contact-form {
+            background-color: rgba(255, 255, 255, 0.582);
+            /* Add a semi-transparent white background */
+            border-radius: 8px;
+            box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+            padding: 20px;
+        }
+
+        .form-group {
+            margin-bottom: 20px;
+        }
+
+        label {
+            display: block;
+            margin-bottom: 5px;
+            color: #555;
+        }
+
+        input[type="text"],
+        input[type="email"],
+        textarea {
```

Comment:
Use this CSS separately to resize only vertically: 
textarea {
resize: vertical;
}